src/js/jobs/components/Item/Status.js

```diff
@@ -26,20 +26,30 @@ export const JobStatus = ({ pad, state }) => {
             </StyledJobStatus>
         );
     }
+    {
+        if (state === "complete") {
+            return (
+                <StyledJobStatus pad={pad}>
+                    <Icon name="check fa-fw" color="green" />
+                    <span>{state}</span>
+                </StyledJobStatus>
+            );
+        }
+    }
 
-    if (state === "complete") {
+    if (state === "error" || state === "cancelled") {
         return (
             <StyledJobStatus pad={pad}>
-                <Icon name="check fa-fw" color="green" />
+                <Icon name="times fa-fw" color="red" />
                 <span>{state}</span>
             </StyledJobStatus>
         );
     }
 
-    if (state === "error" || state === "cancelled" || state === "terminated") {
+    if (state === "terminated") {
         return (
             <StyledJobStatus pad={pad}>
-                <Icon name="times fa-fw" color="red" />
+                <Icon name="fa-solid fa-skull" color="red" />
```

Comment:
```suggestion
                <Icon name="skull fa-fw" color="red" />
```

The `Icon` base component should only need `name="skull"` to get the correct icon. (`fa-fw` specifies that the icon needs to have a fixed width)